tests/Microsoft.Bot.Builder.AI.QnA.Tests/QnAMakerTests.cs

```diff
@@ -838,6 +838,54 @@ public async Task Telemetry_ReturnsAnswer()
             StringAssert.StartsWith(results[0].Source, "Editorial");
         }
 
+        [TestMethod]
+        [TestCategory("AI")]
+        [TestCategory("QnAMaker")]
+        [TestCategory("Telemetry")]
+        public async Task Telemetry_ReturnsAnswer_WhenNoAnswerFoundInKB()
+        {
+            // Arrange
+            var mockHttp = new MockHttpMessageHandler();
+            mockHttp.When(HttpMethod.Post, GetRequestUrl())
+                .Respond("application/json", GetResponse("QnaMaker_ReturnsAnswer _WhenNoAnswerFoundInKb.json"));
```

Comment:
Consider removing extra space from file name for consistency (QnaMaker_ReturnsAnswer_WhenNoAnswerFoundInKb.json instead of QnaMaker_ReturnsAnswer _WhenNoAnswerFoundInKb.json)

Reply:
Wow that is so hard to see.
Fixing it now!